app/assets/javascripts/components/Material.js

```diff
@@ -201,13 +134,36 @@ export default class Material extends Component {
         <NumeralInputWithUnitsCompo
           precision={4}
           value={material.equivalent}
-          disabled={material.reference && material.equivalent}
+          disabled={(material.reference && material.equivalent) !== false}
```

Comment:
```material.reference``` coulde be 1
```material.equivalent```  coulde be 1

```1 && 1 -> 1```

We need boolean for disabled, but sometimes ```{material.reference && material.equivalent}```  returns 1 and React.js will warn you.